libcodechecker/log/option_parser.py

```diff
@@ -153,34 +153,41 @@
     '^-fcse-follow-jumps': 0,
     '^-fcse-skip-blocks': 0,
     '^-ffixed-r2': 0,
+    '^-ffp': 0,
     '^-fgcse-lm': 0,
     '^-fhoist-adjacent-loads': 0,
     '^-findirect-inlining': 0,
     '^-finline-limit.*': 0,
+    '^-finline-local-initialisers': 0,
     '^-fipa-sra': 0,
     '^-fno-aggressive-loop-optimizations': 0,
     '^-fno-delete-null-pointer-checks': 0,
+    '^-fno-jump-table': 0,
     '^-fno-strength-reduce': 0,
     '^-fno-toplevel-reorder': 0,
     '^-fno-unit-at-a-time': 0,
     '^-fno-var-tracking-assignments': 0,
     '^-fpartial-inlining': 0,
     '^-fpeephole2': 0,
+    '^-fr': 0,
     '^-fregmove': 0,
     '^-frename-registers': 0,
     '^-freorder-functions': 0,
     '^-frerun-cse-after-loop': 0,
+    '^-fs': 0,
```

Comment:
It looks like the tests fails. This argument for example matches with "-fsyntax-only".
I suggest adding `$` to the end of short argument names.